src/app/common/components/HeaderComponent.module.scss

```diff
@@ -0,0 +1,3 @@
+.logoPointer {
+    cursor: default !important;
```

Comment:
Not great we need to use `!important` too override these PF styles. Can we file an issue upstream and get to the bottom of why this is necessary?

I think we need to do that before this gets out of control

Reply:
On my end when I remove `!important` from this it still gets applied just fine. But otherwise the solution here would be to add more specificity to the selector. The `cursor: pointer` style is being inherited from `.pf-c-page__header-brand-link` so we could use `.pf-c-page__header-brand-link.logoPointer` to be safe.

Reply:
Oh, weird... with this style of SCSS import I'm not sure how you can even use external (non-compiled) classes in your selectors.

Reply:
Got it, there's a `:global()` syntax for that: https://github.com/konveyor/mig-ui/pull/739

Reply:
@mturley The !important statements in ResultsStep.module.scss are what started my use of !important on all styles relating to patternfly. The .styledCard & .centerCard styles were working locally but were being overridden in the production image.

Reply:
![css_important](https://user-images.githubusercontent.com/11218376/76554897-08331600-646d-11ea-9783-05b62c72c091.gif)